### ISSUE TITLE
Who are the creators of a record?

### ISSUE THREAD

**Issue by chin-rcip** [chin-rcip/collections-model#10]
### Concerning the Creators of the record

For the moment, in the version 1.5 of the Target Model, the pattern for the creation of a record (within a Named Graph) is the following:

`E39_Actor(contributor)->P14_carried_out_by->E65_Creation->P94_has_created->E73_Information_Object(record)`

The contributor is the person from the institution who documented the record (if this information exists) or the contributing museum in the case of an aggregating institution like AiC.

For example, in the case of the record of Jean Paul Riopelle in AiC, the contributing institution is the National Gallery of Canada - Library and Archives.
![CiC_Issue#10-example1](https://user-images.githubusercontent.com/57008591/67962506-7cc39180-fbfd-11e9-8e5a-8ea8f93406c2.png)

But two questions arise from that:
Is the contributing museum really the creator of the record? Should we maybe use another pattern, like with PC14 to type the role of that contributing museum?
Is CHIN also the creator of the record? Technically yes, but then CHIN would appear on every Named Graph, which would be redundant.

### Concerning the source of a record

If a museum documents the sources it uses for recording an entry, should we also document it too? As the dc:bibliographicCitation cannot be used, we need to find a new way of documenting sources. See issue #?

**Comment by KarineLeonardBrouillet** [comment]
I do think it would be important if an institution records a citation for data or information to document it as well. This is especially so for more sensitive collections where the qualifications surrounding an actor or object might be considered highly problematic today. Institutions would most likely appreciate CHIN indicating that X problematic data is not reflective of the institution's current position on the matter.

**Comment by Habennin** [comment]
Why not put the contributing institutions role in the creation as 'contributor'

ie 

E65 -> p01 -> PC14 -> p02 -> E74 Institution
E65 -> p01 -> PC14 -> p14.1 -> E55 Contributor
it would also make sense to include CHIN on every record because in the open world, anyone can use this data anywhere. It provides provenance if the data travels.

**Comment by stephenhart8** [comment]
Indeed, you are right @Habennin everything CHIN does should be mentioned, as we want our data to be used outside CHIN.

Therefor, in the same example, we would have:
![CiC_Issue10-example2](https://user-images.githubusercontent.com/49342191/68689566-8b377480-053e-11ea-828d-fc4fb6a9b88c.png)

**Comment by Habennin** [comment]
looks great to me

**Comment by illip** [comment]
I'm not sure to understand what you are aiming to do. 

1. We have the Named Graph. At this level we will mention the source of the dataset which is the museum. We might also add CHIN as a contributor. To represent the different opinions on a topic we will request different Named Graphs at the same time.

2. We have a specific record within this dataset (E73) which represent, I think, the **initial** record in the museum's database/website. So CHIN isn't a participant at this stage (except if we are ingesting our own records). Two records might give different information and if we have the contributor of this record we will be able to track who said what.

Concerning the pipeline, CHIN will help the institution to map their data. We are going to propose different ideas, but at the end the dataset will represent the museum's vision.

For dct:bibliographicCitation, I think we can use it, I was wrong.

All this to say that I would like to review the purpose of this pattern with you. Do I have the right understanding of the e73 in this pattern? If so, why do we need to mention CHIN each time if we are already described as a contributor at the Named Graph level? Especially, if the e73 represents the initial record and not the new record that we are currently creating for CiC.

**Comment by stephenhart8** [comment]
As discussed on the 29th of November, we need the contributor to be link to the creation event in the role of "Contributor".
CHIN, on the other hand, should not be represented in this creation event, as it will be present in the creation event of the Named Graph, and not the record.

**Comment by illip** [comment]
Decision on issue #14 (named graphs' granularity) might impact the pattern. I would recommend:

1. Mention in the TM how we envision to document museums and CHIN's attributions (pp.21-22 or so)
2. Close this issue if we decide to keep the named graphs as they are.

**Comment by illip** [comment]
Context:

We have decided in Issue #45, to go with one Named Graph per dataset.

The current CHIN's proposal on this issue is:

1. The Named Graph provenance data will include CHIN and Provider X. In most cases, we believe that CHIN will be documented as the "Creator" and Provider X as "Provider".
2. In the [Record Provenance with Aggregated Contributors](https://chin-rcip.github.io/collections-model/target-model/current/general-concepts#record-provenance-with-aggregated-contributors) pattern, as we defined the `E73_Information_Object -> P2_has_type -> E55_Type(Record)` as the original record, CHIN cannot be mentioned unless we ingest our own data.

We will validate this proposal with our Semantic Committee on January 7th.

**Comment by illip** [comment]
All the aforementioned items have been approved by the Semantic Committee on 2021-01-07.

Discussion regarding the update process came up and will be documented in Issue #14.

**Comment by illip** [comment]
> The current CHIN's proposal on this issue is:
> 
> The Named Graph provenance data will include CHIN and Provider X. In most cases, we believe that CHIN will be documented as the "Creator" and Provider X as "Provider".
> In the Record Provenance with Aggregated Contributors pattern, as we defined the E73_Information_Object -> P2_has_type -> E55_Type(Record) as the original record, CHIN cannot be mentioned unless we ingest our own data.

The Named Graph provenance data is now documented in the TM, the Record Provenance is still missing.
